api/src/pcapi/routes/adage_iframe/serialization/educational_institution.py

```diff
@@ -0,0 +1,11 @@
+from pcapi.routes.serialization import BaseModel
+
+
+class EducationalInstitutionWithBudgetResponseModel(BaseModel):
+    id: int
+    name: str
+    institutionType: str | None
```

Comment:
`Institution.institutionType` n'est pas nullable en base de données, donc je pense que ça devrait être la même chose ici : 

```suggestion
    institutionType: str
```